stock_picking_mass_action/i18n/de.po

```diff
@@ -0,0 +1,137 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_picking_mass_action
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-07-21 14:40+0000\n"
+"PO-Revision-Date: 2015-07-21 14:40+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_picking_mass_action
+#: view:stock.picking.mass.action:stock_picking_mass_action.view_stock_picking_mass_action_form
+msgid "Apply"
+msgstr "Apply"
```

Comment:
I don't speak deutch. It will be translated in another PR / by transiflex.

Kind regards.